### ISSUE TITLE
Language issues

### ISSUE THREAD

**Issue by jazzido** [poderopedia/plug-and-play-1.0-RC#2]
Hey,

I just wanted to point out that in order to ease adoption and collaboration from developers outside of Poderopedia, variable names, method names and comments should be in english.

For instance:

`def sugerir_persona()` should be `def suggest_person()`

Just my two cents.

Congratulations on the release!

**Comment by pudo** [comment]
+1

**Comment by rampmaster** [comment]
Is it possible to send a pull request with the changes and proposed name?
Thanks

**Comment by pudo** [comment]
Hm, this affects literally the entire code base and would require a spanish speaker, I think its a larger, strategic decision for the project?

**Comment by jazzido** [comment]
I agree with @pudo, such an effort should come from the mantainers. Almost every file would be modified.

**Comment by miguelpaz** [comment]
@jazzido and @pudo we are on it. We need to map all language names, variable names, method names and comments to change. But since we must start on what adds more value, which are the critical parts that need to be changed according to you?

**Comment by pudo** [comment]
Thanks for the quick response. For contributors, it may be useful to start with file names for quicker navigation as well as the database model, the skeleton of the application.

**Comment by miguelpaz** [comment]
We are on it @pudo and @jazzido. @juaneduardo is changing names as I write this. For broader conversation we have a mailing list https://groups.google.com/forum/?hl=es#!forum/poderopedia-users

**Comment by juaneduardo** [comment]
i changes the functions names in controllers... for the admin application

**Comment by miguelpaz** [comment]
@pudo @jazzido changes were made already. Care to comment or review?

**Comment by vsusarla** [comment]
Hi! I am trying to adopot poderopedia to build a website. I just want to know if there is anyway to launch an english version of the website. I f so what changes need to be made in which files?

**Comment by miguelpaz** [comment]
@vsusarla hey, yes there is. @juaneduardo can help you with that. There is a translation tool for it. We need to add it. Please follow up on this opened issue: https://github.com/poderopedia/plug-and-play-1.0-RC/issues/13

**Comment by vsusarla** [comment]
Thanks a lot @miguelpaz !
